### PR TITLE
[VL] Overwrite murmur3hash and xxhash64 with hash_with_seed

### PR DESCRIPTION
**Pull request by marin-ma**

### CODE REVIEW

**Comment by marin-ma** [comment]
@rui-mo Please pick this commit when update to the latest upstream.

**Comment by rui-mo** [comment]
@marin-ma Thanks!

**Comment by rui-mo** [comment]
@marin-ma Could you help rebase this PR? Thanks.

File: cpp/velox/operators/functions/RegistrationAllFunctions.cc [hunk 1]
```diff
@@ -15,10 +15,12 @@
  * limitations under the License.
  */
 #include "RegistrationAllFunctions.h"
+#include <velox/expression/VectorFunction.h>
```
Comment (rui-mo) [inline cpp/velox/operators/functions/RegistrationAllFunctions.cc:18]: #include "velox/expression/VectorFunction.h"?

**Pull request merged**